drivers/dummy/rasterizer_dummy.h

```diff
@@ -725,20 +725,28 @@ class RasterizerStorageDummy : public RasterizerStorage {
 	}
 
 	bool free(RID p_rid) {
-
 		if (texture_owner.owns(p_rid)) {
 			// delete the texture
 			DummyTexture *texture = texture_owner.get(p_rid);
 			texture_owner.free(p_rid);
 			memdelete(texture);
 		}
-
-		if (mesh_owner.owns(p_rid)) {
+		else if (mesh_owner.owns(p_rid)) {
 			// delete the mesh
 			DummyMesh *mesh = mesh_owner.getornull(p_rid);
 			mesh_owner.free(p_rid);
 			memdelete(mesh);
 		}
+		else if (lightmap_capture_data_owner.owns(p_rid)) {
+			// delete the lightmap
+			LightmapCapture *lightmap_capture = lightmap_capture_data_owner.getornull(p_rid);
+			lightmap_capture_data_owner.free(p_rid);
+			memdelete(lightmap_capture);
+		}
+		else {
```

Comment:
This line is causing the static formatting checks to fail.

We keep the braces on the same line as the "else" statement in the Godot codebase. The same goes for the above "else if" statements.